xsd/netex_part_2/part2_occupancy/netex_oc_occupancy_version.xsd

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns="http://www.netex.org.uk/netex" xmlns:netex="http://www.netex.org.uk/netex" xmlns:siri="http://www.siri.org.uk/siri" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:gml="http://www.opengis.net/gml/3.2" targetNamespace="http://www.netex.org.uk/netex" elementFormDefault="qualified" attributeFormDefault="unqualified" version="1.10" id="netex_occupancy_version">
+	<xsd:include schemaLocation="netex_oc_occupancy_support.xsd"/>
+	<xsd:include schemaLocation="../../netex_framework/netex_utility/netex_units.xsd"/>
+	<xsd:include schemaLocation="../../netex_framework/netex_utility/netex_utility_types.xsd"/>
+	<xsd:include schemaLocation="../../netex_framework/netex_reusableComponents/netex_serviceRestrictions_support.xsd"/>
+	<xsd:include schemaLocation="../../netex_framework/netex_reusableComponents/netex_trainElement_version.xsd"/>
+	<xsd:include schemaLocation="../../netex_framework/netex_reusableComponents/netex_dayType_support.xsd"/>
+	<xsd:include schemaLocation="../../netex_framework/netex_reusableComponents/netex_dayType_version.xsd"/>
+	<xsd:import namespace="http://www.siri.org.uk/siri" schemaLocation="../../siri/siri_base-v2.0.xsd"/>
+	<xsd:group name="OccupancyScopeFilterGroup">
+		<xsd:annotation>
+			<xsd:documentation>The intersection of supplied elements describes the extent that the Occupancy values applies to. (since SIRI 2.1)
+Only vehicle-centric filter (measurement in a part or at an entrance of a TRAIN) are available here, but a stop-centric filtering (measurement in a sector or at a position on a QUAY) can be achieved indirectly via Arrival-/DepartureFormationAssignment.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:group ref="TrainComponentGroup" minOccurs="0"/>
+			<xsd:element name="FareClass" type="FareClassEnumeration" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Fare class in VEHICLE for which occupancy or capacities are specified.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="PassengerCategory" type="siri:NaturalLanguageStringStructure" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Adult, child, wheelchair etc.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+		</xsd:sequence>
+	</xsd:group>
+	<xsd:group name="OccupancyValuesGroup">
+		<xsd:annotation>
+			<xsd:documentation>Occupancy values applying to indicated scope. (since SIRI 2.1)</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element name="OccupancyLevel" type="OccupancyEnumeration" minOccurs="0">
```

Comment:
I would wish clearer definitions of `full` and `OccupancyPercentage`. For trains, coaches, boats: is OccupancyPercentage>=100 <-> "all seats occupied"? For urban buses, metros, etc.: OccupancyPercentage>=100 <-> crushedStandingRoomOnly? OccupancyLevel==full  for coaches if all seats occupied (or would notAcceptingPassengers have to be used?), for all others only after crushed standing room has been consumed? 

Another solution to add clarity might be allowing for multiple OccupancyLevel elements. This could additionally be used, e.g., to distinguish the following two cases for a telecabin: empty && fewSeatsAvailable vs. empty && standingRoomOnly.  

Another question: SBB uses the following occupancy levels for passenger information: *low to average*, *high*, *very high* - would they map to `empty`, `manySeatsAvailable`, `fewSeatsAvailable` respectively? Though *low to average* sounds closer to `manySeatsAvailable`.

Reply:
tried my best. but full is especially tricky. I don't think we can solve it.
OccupancyPercentage added that 100 means full full.  SBB relation I think
1 - many seats available
2 - few seats available
3 - full